protocol/consistencychecks_test.go

```diff
@@ -225,3 +238,118 @@ func TestVerifyKeyLookupResponseWithTB(t *testing.T) {
 		t.Error(e2)
 	}
 }
+
+func TestLookupBeforeMonitoring(t *testing.T) {
+	d, pk := NewTestDirectory(t, true)
+	cc := NewCC(d.LatestSTR(), pk, make(map[string]uint64), true)
+	registerAndVerify(d, cc, alice, key)
+
+	// monitor binding was inserted
+	d.Update()
+
+	// do a lookup before the clock tells us to do monitoring
```

Comment:
It's not entirely clear to me what this test is doing. It's checking whether the binding can be looked up after it's been inserted? I think there should also be a test that checks if `the binding returned to a lookup == the binding returned for monitoring`